ArduCopter/mode.h

```diff
@@ -251,6 +262,15 @@ class Mode {
     void set_throttle_takeoff(void);
     uint16_t get_pilot_speed_dn(void);
 
+    void run_land_controllers(bool pause_descent = false) {
```

Comment:
maybe we should move "run_land_controllers(" up to line 115 so that it is with land_run_horizontal_control() and land_run_vertical_control().  Maybe land_repo_active should move up there too.